doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -0,0 +1,986 @@
+IjXV33XWXnnzyST3wwAP617/+ZT0OIkReXp769u2r7OxsffDBB8R7CSUkJOidd97ReeedpyuuuELLly+3HgkRYvny5erSpYtatmypiRMnEu8l1LJlS02dOlULFy7UNddco7y8POuRECH+9a9/6Ve/+pWefvpp63iPSPQmSoPeLBt6E6VFb5YNvYnSojfLht5EadCbZUNvorTozbKhN1Fa9GbZ0JsoDXqzbOhNlBa9WTb0JkoraL3JDbHQww8/rMcff1x33XWXnnnmGetxEHC7d+/WVVddpezsbE2fPl0XXnih9UgRqVKlSpo8ebKaN2+uzp07a+HChdYjIeAWLlyozp07q3nz5po8ebIqVapkPVJEuuiiizR9+nRlZ2frqquu4p97wGn95S9/0V133aU//elPeuCBB6zHiVj0JkqC3gwPehMlRW+GB72JkqI3w4PeREnQm+FBb6Kk6M3woDdRUvRmeNCbKAl6MzzoTZQUvRke9CZKKoi9yQ2xkCQNGzZM//jHP/TQQw/p4Ycfth4HAbVjxw6lp6dr2bJlmjlzpjp06GA9UkSrUqWKpk2bpnbt2qlbt27KysqyHgkBlZmZqW7duik5OVlTpkzRmWeeaT1SROvQoYMyMzO1cuVKde3aVTk5OdYjIaCefvppPfTQQ3ruuef029/+1nqciEdvojjozfCiN1Fc9GZ40ZsoLnozvOhNFAe9GV70JoqL3gwvehPFRW+GF72J4qA3w4veRHHRm+FFb6K4gtqb3BCLI+688069+OKLeuaZZ3T//feroKDAeiQEyMaNG9WlSxdt2LBBs2fPVtu2ba1HigqVKlXSxIkT1aVLF6Wnp+uDDz6wHgkBM23aNPXo0UNdu3ZVRkaGKleubD1SVGjVqpU++ugjbdy4Ud27d9emTZusR0KAFBQUaOjQoRo2bJhGjRqle++913qkqEFv4lGKa4wAABicSURBVFToTW/QmzgdetMb9CZOhd70Dr2JU6E3vUFv4nToTW/QmzgVetM79CZOhd70Br2J06E3vUFv4lSC3pvcEItj3HrrrXrjjTf0z3/+UwMGDFBeXp71SAiA5cuX69JLL1VeXp5mz56tli1bWo8UVRITE/X222+rb9++uuqqqzRq1CjrkRAQI0eOVK9evdSvXz+NHz9eiYmJ1iNFlXPPPVezZ8/W7t27demll2rFihXWIyEA9u7dq2uvvVYvvviixo4dq5tvvtl6pKhDb+JE6E1v0Zs4GXrTW/QmToTe9B69iROhN71Fb+Jk6E1v0Zs4EXrTe/QmToTe9Ba9iZOhN71Fb+JEIqE3uSEWRQwcOFAzZ87UrFmz1KVLF23ZssV6JBj66KOPlJaWpvr16+vTTz9Vo0aNrEeKSvHx8Xrttdf0f//3f7rttts0dOhQvqs0hjnn9Oijj+qXv/ylhg0bptGjRys+Pt56rKjUuHFjzZ07V+ecc446duyoadOmWY8EQzk5OfrJT36iWbNmafr06br22mutR4pa9CaORm/6g97E0ehN/9CbOBq96R96E0ejN/1Bb+Jo9KZ/6E0cjd70D72Jo9Gb/qA3cTR60z/0Jo4WMb3pgJP48ssvXZMmTVzTpk3dihUrrMeBgVGjRrny5cu7n//8527fvn3W48SM0aNHu/Lly7uBAwe6vXv3Wo8Dn+3Zs8f179/fJSQkuNdee816nJiRl5fnBgwY4BISEtwrr7xiPQ4MLFu2zDVu3Ng1bdrUrVq1ynqcmEFvgt60QW/GNnrTBr0JetMGvQl60wa9GdvoTRv0JuhNG/Qm6E0b9GZsozdt0JuIoN4cy0+IxUmde+65ys7OVp06dXTJJZfo3XfftR4JPtm/f7/uuOMODRkyRA8++KDefPNNfrS8j2666SZlZGRo+vTpSklJ0dq1a61Hgk++/vprpaSkaMaMGcrIyNAvfvEL65FiRoUKFTR27Fj9+te/1i233KK77rpLBw4csB4LPnn77bfVsWNH1a9fX9nZ2WrRooX1SDGD3oxd9KYtejN20Zt26M3YRm/aoTdjF71pi96MXfSmHXozttGbdujN2EVv2qI3Yxe9aYfejG2R1pvcEItTqlOnjmbPnq0777xT1157rYYOHaqDBw9ajwUPfffdd+rSpYvefPNNjRs3Tn/84x8VCoWsx4o53bp10xdffKHy5cvrggsu0MSJE61HgscyMjLUvn17hUIhzZ8/X127drUeKeaEQiE98cQTmjBhgt58802lpqZq/fr11mPBQ/n5+Xr44Yc1YMAAXXfddZo5c6Zq165tPVbMoTdjD70ZDPRm7KE37dGbsYfeDAZ6M/bQm8FAb8YeetMevRl76M1goDdjD70ZDPRm7KE37dGbsSdie9P6Z9QicowePdpVqFDBXXHFFW7Tpk3W48ADM2bMcHXq1HHnnXee+/LLL63HgTv84/5vuOEGFxcX5/7whz+4Q4cOWY+EMDt06JD7/e9/7+Li4twvfvEL/lmPgFi+fLlr2bKlq1u3rps5c6b1OPDA999/77p06eIqVqzo/v3vf1uPg/+hN6MfvRk89Gb0ozeDid6MfvRmMNGb0Y/eDB56M/rRm8FEb0Y/ejOY6M3oR28GD70Z/ejNYKI3o18E9+ZYbohFiSxcuNA1adLE1alTx02ePNl6HITJ/v373QMPPODi4uJc//793c6dO61HwnGGDx/uEhMT3eWXX+7Wr19vPQ7CZN26dS4tLc1VqFDB/fOf/7QeB8fJzc1111xzjYuLi3MPP/ywO3DggPVICJOJEye62rVru2bNmrlFixZZj4Pj0JvRid4MPnozOtGbwUZvRi96M9jozehEbwYfvRmd6M1gozejF70ZbPRmdKI3g4/ejE70ZrDRm9ErwnuTG2JRcjt37nS33Xabk+QGDRrkdu/ebT0SyuDLL790F110katYsaL761//aj0OTmHZsmWubdu2rkqVKu7111+3HgdlNH78eFe9enXXqlWrSAyImPLaa6+5ypUru3bt2rmVK1
```

Comment:
```suggestion
    "The parallel version of MDAnalysis is still under development. For existing solutions and some implementations of parallel analysis, go to [PMDA](https://www.mdanalysis.org/pmda/). PMDA (<a data-cite=\"fan_pmda_2019\" href=\"https://doi.org/10.25080/Majora-7ddc1dd1-013\">Shujie *et al.*, 2019</a>) applies the aforementioned split-apply-combine scheme with Dask. In the future, it may provide a framework that consolidates all the parallelisation schemes described in this tutorial."
```

Reply:
This is about the last parallel section part.